src/Common/src/TypeSystem/Interop/IL/NativeStructType.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace Internal.TypeSystem.Interop
 {
-    public partial class NativeStructType : MetadataType
+    public partial class NativeStructType : MetadataType, IPrefixMangledType
```

Comment:
Can you put this in a separate `.Mangling.cs` dot file like the others?

This file is not included in the JIT codebase right now, so this is not a build break, but I think it will get used at some point because we do want to do struct marshalling in jitted code. `IPrefixMangledType` doesn't exist in `System.Private.Jit` because we don't need mangled linkage names when jitting.